pkg/test/mocks/repository_config_dao.go

```diff
@@ -84,3 +86,12 @@ func (r *RepositoryConfigDao) ValidateParameters(orgId string, req api.Repositor
 	r.Called(orgId, req)
 	return api.RepositoryValidationResponse{}, nil
 }
+
+func NewRepositoryConfigDao(t *testing.T) *RepositoryConfigDao {
+	m := &RepositoryConfigDao{}
+	m.Mock.Test(t)
+
+	t.Cleanup(func() { m.AssertExpectations(t) })
```

Comment:
This makes it so we don't have to call AssertExpectations in our handler tests, which in a couple of test cases we were forgetting to do. `Cleanup(func())` is run at the end of the test, but before it ends.

The mockery generator was already doing this. I just added it here for repository configs.